### PR TITLE
feat(wiki): cdk

### LINKED ISSUE #13: Initialize CDK Content
## Context

This task concentrates on initializing the CDK content for our Wiki.

Polygon Chain Development Kit (CDK) is the next-generation successor to the existing Supernets product suite. It is also a critical component of Polygon 2.0. For more details about the current Supernets product suite, please refer to our Edge client-based Supernets documentation [here](https://wiki.polygon.technology/docs/supernets).

The CDK framework employs zkValidium, a technology that expands on the implementation of Polygon's zkEVM, with the addition of a data availability layer overseen by a Data Availability Committee (DAC).

## Done Criteria

- [x] Create a temporary new entry on the Navbar for Polygon 2.0 (the design is subject to future changes).
- [x] Establish a new CDK section in the sidebars.js.
- [x] Craft a "What are CDK" overview, highlighting the essence and capabilities of zkSupernets.
    - [x] Provide a brief explanation of what CDK are and their functionality.
    - [x] Provide contextual section on app-chains, data availability, and validiums.
    - [x] Introduce a "Why CDK" section.
    - [x] Highlight the CDK program.
- [x] Produce a system attributes doc that distinctly convey the functionality.
- [x] Produce a technical overview of the DAC.
   - [x] Define data availability and the role of DACs.
   - [x] Indicate that the CDK is essentially the zkEVM tied together with an additional data availability layer (DAC).
   - [x] Include a breakdown of data flow.
   - [x] Include an overview of zkEVM and outline that CDK use the same implementation.
   - [x] Compare with zkEVM.
- [x] Generate a standalone document dedicated to the CDK program.
- [x] Produce an index page for CDK.
- [x] Produce a quickstart guide to launch a simple test (CDK-developed) network.
- [ ] Ensure associated repository READMEs are updated.

## Additional Information

The primary aim of these initial documents is to serve as an accessible starting point for a diverse range of audiences, facilitating their understanding of Polygon CDK.

We also plan to follow this with a new set of product documentation that will diverge from the existing Supernets documentation.

### PR DESCRIPTION
**Pull request by DannyS03**
## Context

This pull request introduces the initial batch of content on the Polygon CDK to the Polygon Wiki.

### Details

- Addresses and resolves #13, adhering to the specified criteria.
- **Key Additions to CDK Content:**
    - **Introduction to CDK**: A new document explaining what the CDK is.
    - **Validium's System Attributes**: A guide that provides a snapshot of the system attributes of the Validium.
    - **DA Layer Overview**: A document providing an overview of the Data Availability layer.
    - **Polygon Labs Support Program**: Information on the support program offered by Polygon Labs.
    - **Deployment Differences with zkEVM**: A comparative guide on deployment variations with zkEVM.
    - **Quickstart Guide**: A step-by-step guide to quickly spin up a validium deployment, courtesy of Snapchain.
- **Enhancements:**
    - Improved landing and index pages for better user experience.
    - Updated the navigation bar for easier access to resources.
    - Docusaurus-based configs have been updated to reflect the CDK content.

### CODE REVIEW

**Review by EmpieichO — APPROVED** [review]

File: docs/cdk/validium-attributes.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+### Metrics Overview
+
+| Metric              | Measurement        | Measurement Context   | Description                                                            | Key Term Definition |
+|---------------------|--------------------|-----------------------|------------------------------------------------------------------------|---------------------|
+| Unified Liquidity   | High               | Polygon 2.0 | Seamlessly interacts across chains via LXLY Bridge for high liquidity  | Seamless flow of assets between different chains. |
+| Future-Proof        | Essential          | Polygon 2.0 | Key component of the Polygon 2.0 framework for future-ready operations | Preparation and composability of the product suite for future advancements. |
+| Gas Availability    | Unlimited | L2 | Supports virtually unlimited processing capacity due to unlimited L2 gas | The amount of computational effort required to process transactions. |
+| Transaction Fees    | Very Low           | L2 | Designed for efficiency, resulting in significantly reduced fees due to DAC       | Costs associated with executing transactions. |
```
Comment (arnaubennassar) [inline docs/cdk/validium-attributes.md:29]: in significantly reduced fees due to ~DAC~ off-chain DA and ZKPs

File: docs/cdk/deployment-differences.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+   - [Repository Link](https://github.com/0xPolygon/cdk-validium-node)
+2. **zkEVM Contracts**: These smart contracts facilitate zkEVM operations on Ethereum.
+   - Components: `PolygonZkEVMBridge`, `PolygonZkEVMGlobalExitRoot`, and others. A full breakdown is available [here](https://wiki.polygon.technology/docs/zkevm/architecture/).
+   - [Repository Link](https://github.com/0xPolygonHermez/zkevm-contracts)
+
+### Validium Deployment
+
+1. **Data Availability Layer**: The primary distinction of the validium. It ensures off-chain data availability while only storing the hash of transaction data on L1.
+   - Components: zkEVM components + PostgreSQL database (with plans to transition to a key-value store in the near future.)
+   - [Repository Link](https://github.com/0xPolygon/supernets2-data-availability)
```
Comment (arnaubennassar) [inline docs/cdk/deployment-differences.md:36]: https://github.com/0xPolygon/cdk-data-availability

File: docs/cdk/deployment-differences.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Polygon CDK's validium offering is a unique scaling solution that builds upon the foundation of the zkEVM. While it inherits the core functionalities of the zkEVM, it introduces a distinct approach to data availability by incorporating the [DAC](/docs/cdk/dac.md). This guide will delve into the nuances of deploying both systems, highlighting the key differences.
+
+## Deployment Differences
+
+### zkEVM Deployment
+
+1. **zkEVM Node**: This node manages the Polygon zkEVM Network, processing transactions, maintaining state, and interacting with Ethereum.
+   - Components: JSON RPC, Pool DB, Sequencer, Etherman, Synchronizer, State DB, Aggregator, Prover.
+   - [Repository Link](https://github.com/0xPolygon/cdk-validium-node)
```
Comment (arnaubennassar) [inline docs/cdk/deployment-differences.md:27]: https://github.com/0xPolygon/zkevm-node

File: docs/cdk/deployment-differences.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+2. **zkEVM Contracts**: These smart contracts facilitate zkEVM operations on Ethereum.
+   - Components: `PolygonZkEVMBridge`, `PolygonZkEVMGlobalExitRoot`, and others. A full breakdown is available [here](https://wiki.polygon.technology/docs/zkevm/architecture/).
```
Comment (arnaubennassar) [inline docs/cdk/deployment-differences.md:29]: `PolygonZkEVM` main smart contract (rollup)

File: docs/cdk/deployment-differences.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+   - [Repository Link](https://github.com/0xPolygonHermez/zkevm-contracts)
+
+### Validium Deployment
+
+1. **Data Availability Layer**: The primary distinction of the validium. It ensures off-chain data availability while only storing the hash of transaction data on L1.
+   - Components: zkEVM components + PostgreSQL database (with plans to transition to a key-value store in the near future.)
+   - [Repository Link](https://github.com/0xPolygon/supernets2-data-availability)
+2. **zkEVM Node with Validium Extensions**: The node is extended to support the data availability layer.
```
Comment (arnaubennassar) [inline docs/cdk/deployment-differences.md:37]: https://github.com/0xPolygon/cdk-validium-node

File: docs/cdk/deployment-differences.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+3. **Validium-specific DAC Contract**: One additional contract handles interactions with the DAC and data availability layer.
+   - Component(s): `Supernets2DataCommittee.sol`
```
Comment (arnaubennassar) [inline docs/cdk/deployment-differences.md:39]: `CDKDataCommittee.sol`, `CDKValidium.sol`

File: docs/cdk/deployment-differences.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+   - [Repository Link](https://github.com/0xPolygon/supernets2-contracts)
```
Comment (arnaubennassar) [inline docs/cdk/deployment-differences.md:40]: https://github.com/0xPolygon/cdk-validium-contracts

File: docs/cdk/deployment-differences.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+| **Additional Contracts** | None                                                                                    | Validium-specific DAC contract                               |
+| **Infrastructure Needs** | Standard infrastructure                                                                 | Dedicated infrastructure for data availability layer and DACs                                |
+
+> **JSON RPC, Pool DB, Sequencer, Etherman, Synchronizer, State DB, Aggregator, Prover
+
+### Transaction Flow in Validium
+
+Unlike zkEVM, where all transaction data is published on L1, Validium only publishes the hash of the transaction data. This hash, termed the _Accumulated Input Hash_, must be approved by a majority of the DAC members. The Sequencer sends both the hash and the transaction data to the DAC for verification. Once approved, the hash, along with the signatures from the DAC members, is sent to the Consensus L1 contract of the Validium protocol. After verification, the hash and the ZK-proof are added to the L1 State, forming the _Consolidated State_.
+
+## Deployment Steps
```
Comment (arnaubennassar) [inline docs/cdk/deployment-differences.md:58]: We need a dedicated guide for this. Linking to the different repos it's not enough

File: docs/cdk/validium-attributes.md (hunk 1 above)
Comment (DannyS03) [reply docs/cdk/validium-attributes.md:29]: Addressing in follow-up PR.

File: docs/cdk/deployment-differences.md (hunk 6 above)
Comment (DannyS03) [reply docs/cdk/deployment-differences.md:39]: Addressing in follow-up PR

File: docs/cdk/deployment-differences.md (hunk 8 above)
Comment (DannyS03) [reply docs/cdk/deployment-differences.md:58]: Agreed, and work is currently underway. Based on the team(s) requests, the goal of this initial batch of docs was to offer a streamlined onboarding experience, with an emphasis on exposure through a Quickstart guide. For this reason, we are presently opting for external references. I will make sure to clarify this point in the docs.

**Pull request merged**